x-pack/plugins/ml/common/types/capabilities.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { KibanaRequest } from 'kibana/server';
+
+export const userMlCapabilities = {
+  // Anomaly Detection
+  canGetJobs: false,
+  canGetDatafeeds: false,
+  // Calendars
+  canGetCalendars: false,
+  // File Data Visualizer
+  canFindFileStructure: false,
+  // Filters
+  canGetFilters: false,
+  // Data Frame Analytics
+  canGetDataFrameAnalytics: false,
```

Comment:
If I log in as a user with the `machine_learning_user` role. the clone item is showing for data frame analytics jobs:

![image](https://user-images.githubusercontent.com/7405507/80007737-9b288c80-84be-11ea-8485-d363b0b756bc.png)

Reply:
This is a bug in master. I think it is outside the scope of this PR to fix here.
I've raised https://github.com/elastic/kibana/issues/64235 to cover it.